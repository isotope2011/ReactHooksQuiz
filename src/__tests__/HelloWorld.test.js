import React from 'react';
import { render } from '@testing-library/react';

import HelloWorld from '../views/HelloWorld';

describe('HelloWorld View Component', () => {
    test('Show Hello World text', () => {
        const { getByText } = render(<HelloWorld />);
        expect(getByText('Hello World!!')).toMatchSnapshot();
    })
});
