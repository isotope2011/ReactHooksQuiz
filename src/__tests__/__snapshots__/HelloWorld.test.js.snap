// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld View Component Show Hello World text 1`] = `
<h2>
  Hello World!!
</h2>
`;
